distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1424,54 +1447,244 @@ public void testProductDownloadListEmptyWhenNoDownloads() {
+     */
+    @Test
+    public void testProductDownloadListWithTwoActiveDownloads() throws IOException {
+
+        String filename1 = "product1.txt";
+        String metacardId1 = generateUniqueMetacardId();
+        String resourceData1 = getResourceData(metacardId1);
+
+        String filename2 = "product2.txt";
+        String metacardId2 = generateUniqueMetacardId();
+        String resourceData2 = getResourceData(metacardId2);
+
+        int delayBetweenRetries = 200;
+
+        String startDownloadUrl1 = setUpDownloadResponse(metacardId1, filename1, resourceData1, NO_RETRIES, delayBetweenRetries);
+        String startDownloadUrl2 = setUpDownloadResponse(metacardId2, filename2, resourceData2, NO_RETRIES, delayBetweenRetries);
+
+        // @formatter:off
+        String downloadId1 = when().get(startDownloadUrl1).then().log().all()
+                .extract().jsonPath().getString("downloadId");
+        // @formatter:on
+        assertThat(downloadId1, not(isEmptyString()));
+
+        // @formatter:off
+        String downloadId2 = when().get(startDownloadUrl2).then().log().all()
+                .extract().jsonPath().getString("downloadId");
+        // @formatter:on
+        assertThat(downloadId2, not(isEmptyString()));
+
+        String getAllDownloadsUrl = RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl();
+
+        // @formatter:off
+        List<Map<String, Object>> downloads =
+                expect("List of active downloads is not empty").within(30, SECONDS)
+                        .until(() -> when().get(getAllDownloadsUrl).then().log().all()
+                                .extract().body().jsonPath().getList(""), hasSize(2))
+                        .lastResult();
+        // @formatter:on
+
+        Map download1 = downloads.get(0);
+        assertThat(download1.size(), is(6));
+        Map download2 = downloads.get(1);
+        assertThat(download2.size(), is(6));
+
+        if (download1.get("downloadId")
+                .equals(downloadId1)) {
+            checkDownloadInProgress(download1, downloadId1, filename1, resourceData1);
+            checkDownloadInProgress(download2, downloadId2, filename2, resourceData2);
+
+        } else if (download1.get("downloadId")
+                .equals(downloadId2)) {
+            checkDownloadInProgress(download1, downloadId2, filename2, resourceData2);
+            checkDownloadInProgress(download2, downloadId1, filename1, resourceData1);
+        } else {
+            LOGGER.error("Unexpected data in the download");
+            fail("Unexpected data in the download");
+        }
+
+    }
+
+    /**
+     * Determines that when two downloads are downloaded at once and one fails, it does not affect the
+     * success of the other download.
+     * @throws Exception
+     */
+    @Test
+    public void testProductDownloadListWithTwoActiveDownloadsOneFails() throws Exception {
+
+        cometDClient = setupCometDClient(Arrays.asList(NOTIFICATIONS_CHANNEL, ACTIVITIES_CHANNEL));
```

Comment:
We should be able to only listen to the `ACTIVITIES_CHANNEL` since it receives both started and completed events.